@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1617,7 +1653,8 @@ export class TextElementsRenderer {
             textDistance,
             this.m_viewState.lookAtDistance
         );
-        const iconReady = renderIcon && poiRenderer.prepareRender(pointLabel, this.m_viewState.env);
+        const iconReady =
+            renderIcon && poiRenderer?.prepareRender(pointLabel, this.m_viewState.env);
```

Comment:
```suggestion
        const iconReady =
            renderIcon && poiRenderer?.prepareRender(pointLabel, this.m_viewState.env) === true;
```
Better to return a simple boolean instead of something that can be undefined.

Reply:
done